website/templates/project/contributors.mako

```diff
@@ -280,6 +334,64 @@
     </tr>
 </script>
 
+<script id="accessRequestRow" type="text/html">
+    <tr>
+        <td data-bind="attr: {class: accessRequest.expanded() ? 'expanded' : null,
+                       role: $root.collapsed() ? 'button' : null},
+                       click: $root.collapsed() ? toggleExpand : null">
+            <img data-bind="attr: {src: accessRequest.user.profile_image_url}" />
+            <span data-bind="attr: {class: accessRequest.expanded() ? 'fa toggle-icon fa-angle-up' : 'fa toggle-icon fa-angle-down'}"></span>
+            <div class="card-header">
```

Comment:
Is it worth splitting this div into its own piece that can be used in both here and the contribRow?

Reply:
Oh man, yes, you made me realize that there was a bunch of unnecessary code here having to do with unregistered contributors and/or searching. Got rid of that, much smaller now!